src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -136,7 +130,31 @@ case class MethodConfiguration(
                    methodName: String,
                    @(ApiModelProperty@field)(required = true, value = "The version of method from method store")
                    @(VertexProperty@field)
-                   methodVersion: String,
+                   methodVersion: String
+                   )
+@ApiModel(value = "Method Store Configuration")
+case class MethodStoreConfiguration(
+                   @(ApiModelProperty@field)(required = true, value = "The namespace of method config from method store")
+                   @(VertexProperty@field)
```

Comment:
I don't think the new case classes use the reflection stuff to populate vertexes so the VertexProperty annotation is not required.